pkgs/games/fteqw/generic.nix

```
@@ -0,0 +1,55 @@
+{ pname
+, releaseFile
+, buildFlags
+, buildInputs
+, nativeBuildInputs ? []
+, postFixup ? ""
+, description
+, ... }@args:
+
+with args;
+
+stdenv.mkDerivation {
+  inherit pname buildFlags buildInputs nativeBuildInputs postFixup;
```

```suggestion
  inherit (args) pname buildFlags buildInputs nativeBuildInputs postFixup;
```
or maybe this already fixes it?